docs/wiki/framework/hunterkiller-framework.md

```diff
@@ -38,6 +39,7 @@ class CfgVehicles {
         ace_hunterkiller[] = {{{0}, 1}, {{1}, 2}}; // e.g. vehicle where commander is [1] instead of [0,0]
     };
 };
+{{ endraw }}
 ```
```

Comment:
```suggestion
```
{% endraw %}
```

Reply:
Well this suggestion didn't work, but `{% endraw %}` should be after closing code block.